tensorflow_examples/models/custom_resnet/resnet.py

```diff
@@ -0,0 +1,58 @@
+class ResNet:
+    
+    @staticmethod
```

Comment:
Avoid using staticmethod

http://google.github.io/styleguide/pyguide.html#217-function-and-method-decorators

Reply:
Yes, I have removed the decorator. This will reflect in the next commit. I will make sure all code will follows the guidelines.